core/src/tile/tileWorker.cpp

```diff
@@ -94,10 +97,10 @@ void TileWorker::run() {
 
             // float loadTime = (float(clock() - begin) / CLOCKS_PER_SEC) * 1000;
             // LOG("loadTime %s - %f", task->tile->getID().toString().c_str(), loadTime);
-
-            m_tileManager.tileProcessed(std::move(task));
```

Comment:
Not sure what you mean by that. The tile state logic is certainly not perfect. That's why I started the rather large rework to bring it into a shape that shall be easier to reason about.